src/ui/pages/SwapForm/fieldsets/FiatInputValue/FiatInputValue.tsx

```diff
@@ -3,19 +3,28 @@ import type { SwapFormView } from '@zeriontech/transactions';
 import BigNumber from 'bignumber.js';
 import React from 'react';
 import { formatCurrencyValue } from 'src/shared/units/formatCurrencyValue';
-import { formatPercent } from 'src/shared/units/formatPercent/formatPercent';
 import { UIText } from 'src/ui/ui-kit/UIText';
 import { isNumeric } from 'src/shared/isNumeric';
 import { useCurrency } from 'src/modules/currency/useCurrency';
-import { exceedsPriceImpactThreshold } from '../../shared/price-impact';
+import type { PriceImpact } from '../../shared/price-impact';
+
+interface FiatInputValueProps {
```

Comment:
I can see potential merge conflicts with bridge form. We should merge this carefully